<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE xml>
<sdf version="1.9">
  <model name="bad_veg">
    <link name="body">
      <visual name="body_visual">
        <geometry>
          <box>
            <size>0.5 0.5 0.5</size>
          </box>
        </geometry>
        <material>
          <ambient>1 0 0 1</ambient>
          <diffuse>1 0 0 1</diffuse>
        </material>
      </visual>
    </link>
  </model>
</sdf>
